scwx-qt/source/scwx/qt/types/qt_types.cpp

```diff
@@ -15,20 +15,23 @@ static const std::unordered_map<UiStyle, std::string> qtStyleName_ {
    {UiStyle::Fusion, "Fusion"},
    {UiStyle::FusionLight, "Fusion"},
    {UiStyle::FusionDark, "Fusion"},
+   {UiStyle::FusionQt6Ct, "Fusion"},
    {UiStyle::Unknown, "?"}};
 
 static const std::unordered_map<UiStyle, std::string> uiStyleName_ {
    {UiStyle::Default, "Default"},
    {UiStyle::Fusion, "Fusion"},
    {UiStyle::FusionLight, "Fusion Light"},
    {UiStyle::FusionDark, "Fusion Dark"},
+   {UiStyle::FusionQt6Ct, "Fusion with qt6ct Palette"},
```

Comment:
Two different ideas here.

1. Add each qt6ct palette as a new Fusion <palette> UiStyle enum (e.g., "Fusion Airy", "Fusion Darker", etc.). I'm not sure most users would be differentiating a style vs. a palette.
2. Add a separate palette selection (e.g., "Light", "Dark", "Airy", "Darker", etc.). This would only be applicable if the theme is Fusion (or maybe not?). This could allow custom palettes, but would also introduce more complexity, in both the software and what the user manages.

Thoughts?

Reply:
I think 1 is the cleaner way to do it. I would personally like the ability to do my own styles, but I do not know how common that use case would be, given that the file format is specific to qt6ct. If you are OK with me implementing it, I would just add a "Fusion Custom" and a file selector. It would not be that hard to do.

Reply:
Sounds good to me.